client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -202,47 +201,55 @@ class TransferOtherUser extends React.Component {
 				onClose={ this.handleDialogClose }
 			>
 				<h1>{ this.props.translate( 'Confirm Transfer' ) }</h1>
-				<p>
-					{ this.props.translate(
-						'Do you want to transfer the ownership of {{strong}}%(domainName)s{{/strong}} ' +
-							'to {{strong}}%(selectedUserDisplay)s{{/strong}}?',
-						{
-							args: { domainName, selectedUserDisplay },
-							components: { strong: <strong /> },
-						}
-					) }
-				</p>
+				<p>{ this.getDialogMessage() }</p>
 			</Dialog>
 		);
 	}
 
+	getDialogMessage() {
+		const { selectedDomainName: domainName, isMapping, translate } = this.props;
+		const selectedUserDisplay = this.getSelectedUserDisplayName();
+
+		if ( isMapping ) {
+			return translate(
+				'Do you want to transfer the domain mapping of {{strong}}%(domainName)s{{/strong}} ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Do you want to transfer the ownership of {{strong}}%(domainName)s{{/strong}} to {{strong}}%(selectedUserDisplay)s{{/strong}}?' )` *ES Score: 8*